crates/cibse_guide_e/src/chapter_10/equation_10_10.rs

```diff
@@ -0,0 +1,24 @@
+pub mod openfire_runner;
+
+pub fn limiting_velocity(g: f64, h: f64, t_f: f64, t_0: f64) -> f64 {
+    return 0.64 * (g * h  * (t_f - t_0)/t_f).powf(0.5);
+}
+
+pub fn limiting_velocity_symbols(v_e: String, g: String, h: String, t_f: String, t_0: String) -> String {
```

Comment:
```suggestion
pub fn equation(v_e: String, g: String, h: String, t_f: String, t_0: String) -> String {
```

This is not just used for symbols, it is used to substitute values in general. I suggest the following name change. Again, just so that we can be consistent throughout the whole code base when we do this.